### PR TITLE
add amplitude parameter to sonify.pitch_contour

### PR DESCRIPTION
**Pull request by rabitt**
Adds an optional `amplitude` parameter to `sonify.pitch_contour` that applies an amplitude envelope to the pitch contour.

### CODE REVIEW

**Comment by craffel** [comment]
Thanks @rabitt !  I am traveling until Monday, so if someone else with push access wants to CR before then feel free.  Looks like for now at least the test needs to be updated to fix Travis.

File: tests/test_sonify.py [hunk 1]
```diff
@@ -95,3 +96,9 @@ def test_pitch_contour():
     x = mir_eval.sonify.pitch_contour(times + 5.0, freqs, fs, length=fs * 7)
     assert len(x) == fs * 7
     assert np.allclose(x[:fs * 5], x[0])
+
+    # Test with explicit amplitude
+    x = mir_eval.sonify.pitch_contour(times, freqs, fs, length=fs * 7,
+                                      amplitude=amps)
```
Comment (ejhumphrey) [inline tests/test_sonify.py]: does this work? isn't the kwarg named `amplitudes` (plural)?

**Comment by rabitt** [comment]
Hmmm...I think something in Travis unrelated (?) to this change is [broken](https://travis-ci.org/rabitt/mir_eval/jobs/298700712#L2016-L2030)... @craffel any ideas?

**Comment by craffel** [comment]
My guess is that this is due to a breaking Matplotlib release.  I will create a separate issue.

**Comment by craffel** [comment]
Testing issue should be resolved via #268 thanks to @bmcfee .

**Comment by rabitt** [comment]
@ejhumphrey @craffel I think this is good to go?

File: mir_eval/sonify.py [hunk 2]
```diff
@@ -233,8 +237,17 @@ def pitch_contour(times, frequencies, fs, function=np.sin, length=None,
     # Estimate frequency at sample points
     f_est = f_interp(np.arange(length))
 
+    if amplitudes is None:
+        a_est = np.ones((length, ))
+    else:
+        # build an amplitude interpolator
+        a_interp = interp1d(
+            times * fs, amplitudes, kind=kind,
```
Comment (craffel) [inline mir_eval/sonify.py]: Can you change the docstring for `kind` so that it reads
> Interpolation mode for the frequency and amplitude values.

**Comment by craffel** [comment]
Thanks!

**Pull request merged**